src/map/Map.ts

```diff
@@ -0,0 +1,123 @@
+import Phaser from "phaser";
+
+const ROOT_MAP_FOLDER = 'map';
+
+const TILE_WIDTH = 32;
+const TILE_HEIGHT = 32;
+
+const TILE_MARGIN = 0;
+const TILE_SPACING = 0;
+
+const MAP_KEY = ROOT_MAP_FOLDER;
```

Comment:
these two consts have the exact same value... why?
- MAP_KEY
- ROOT_MAP_FOLDER